bundles/nl.esi.comma.assertthat/src/nl/esi/comma/assertthat/AssertThat.xtext

```diff
@@ -18,23 +18,12 @@ GenericScriptBlock:
 
 ScriptParametersCustom:
     '{'
-        'using-path' ':'   scriptPath = STRING
-        ('with-args' ':' '{'
-            (argType+=SCRIPT_PARAM_TYPE argVal+=JsonElements) 
-            (',' argType+=SCRIPT_PARAM_TYPE argVal+=JsonElements)?
-        '}')?
+        ('using-api'   ':'  scriptApi = STRING)
+        ('with-args'   '{' (scriptArgs+=Expression) (',' scriptArgs+=Expression)* '}')?
+        ('output-file' ':'  scriptOut = STRING) 
     '}' 
 ;
 
-enum SCRIPT_PARAM_TYPE:
-    OUTPUT='OUTPUT'
-    | FILE='FILE'
-    | FILE_REF='FILE_REF'
-    | VALUE='VALUE'
-    | VALUE_REF='VALUE_REF'
-    | NAME_ONLY='NAME_ONLY'
-    ;
-
 enum MARGIN_TYPE:
     ABSOLUTE='Absolute'
```

Comment:
Please make all the values of this enum also lower case, thus **not** starting with a capital.

```suggestion
    ABSOLUTE='absolute'
```